app/components/marketing/events-section.tsx

```diff
@@ -1,41 +1,53 @@
 import type { Event, Group } from '@prisma/client';
 
-type EventsListProps = {
-  events: Event[];
-  groups: Group[];
-};
+interface ExtendedEvent extends Event {
+  group: Group;
+}
 
-export function EventsSection({ events, groups }: EventsListProps) {
+export function EventsSection({ events }: { events: ExtendedEvent[] }) {
   return (
     <section className="bg-secondary">
       <div className="sm:ml-24 pt-10 pb-20 space-y-3 ">
         <h2 className="text-center sm:text-left sm:text-5xl text-4xl font-bold italic text-primary">Upcoming Events</h2>
         <div className="flex flex-wrap gap-6 justify-center sm:justify-start">
           {events.map((event, index) => {
-            const group = groups.find((group) => group.id === event.groupId);
-            if (!group) return null;
-            return <EventCard key={event.id} event={event} group={group} index={index} />;
+            return <EventCard key={event.id} event={event} index={index} />;
           })}
         </div>
       </div>
     </section>
   );
 }
 
-export function EventCard({ event, group, index }: { event: Event; group: Group; index: number }) {
+export function EventCard({ event, index }: { event: ExtendedEvent; index: number }) {
+  const date = event.date.toLocaleString('en-US', {
+    weekday: 'short',
+    month: 'short',
+    day: 'numeric',
+    hour: 'numeric',
+    minute: 'numeric',
+    hour12: true,
+    timeZone: 'America/Los_Angeles',
+    timeZoneName: 'short',
+  });
+  const parts = date.split(', ');
+  const formattedTime = `${parts[0]}, ${parts[1]} - ${parts[2]}`;
+
   return (
     <div className="flex flex-col space-y-3 w-[350px] mx-6 sm:mx-0 max-w-md p-6 rounded-2xl bg-primary">
-      <img
-        className={`${index === 1 && 'sm:order-2'} pt-8 w-full object-contain box-border`}
-        src={event.imgUrl}
-        alt={event.imgAlt}
-      />
+      {event.imgUrl && event.imgAlt && (
```

Comment:
> Although not best for accessibility, I think I would rather show an image even if there is no alt text.
> 
> My thought is that it'd be a worse user experience to not see any image at all if an image does exist. This is probably debatable so please don't change unless other team members feel the same way.

Agree, that this is a matter for discussion. No doubt that always having both (src and alt) is the best. It's been discussed (when choosing accessible components like Headless UI) and I think we should **prioritize the accessibility** since we're creating a social web app. It seems that the suggested idea (Andre's maybe) allows us to keep both attributes. 
As I understood, the idea under this approach is that we can provide a user with a relevant imgUrl and imgAlt by default. As a consequence, we'll always see our cards or other UI filled with images and alternative text. Not sure if it's a good UX to make such decisions for the user but I can leave it as it is until better ideas come)